### ISSUE TITLE
Seems to be broken with newer versions of Rails.

### ISSUE THREAD

**Issue by jash** [TylerRick/thin_out_backups#1]
Got this exception when attempting to compile assets:

```
SyntaxError: /usr/local/bundle/gems/quality_extensions-1.4.0/lib/quality_extensions/module/attribute_accessors.rb:9: `@@{' is not allowed as a class variable name
/usr/local/bundle/gems/quality_extensions-1.4.0/lib/quality_extensions/module/attribute_accessors.rb:9: syntax error, unexpected end-of-input
        unless defined? @@{:instance_writer=>false, :in...
                          ^
/usr/local/bundle/gems/quality_extensions-1.4.0/lib/quality_extensions/module/attribute_accessors.rb:8:in `class_eval'
/usr/local/bundle/gems/quality_extensions-1.4.0/lib/quality_extensions/module/attribute_accessors.rb:8:in `block in mattr_reader'
/usr/local/bundle/gems/quality_extensions-1.4.0/lib/quality_extensions/module/attribute_accessors.rb:7:in `each'
/usr/local/bundle/gems/quality_extensions-1.4.0/lib/quality_extensions/module/attribute_accessors.rb:7:in `mattr_reader'
/usr/local/bundle/gems/quality_extensions-1.4.0/lib/quality_extensions/module/attribute_accessors.rb:45:in `mattr_accessor'
/usr/local/bundle/gems/actionview-4.2.1/lib/action_view/helpers/form_helper.rb:1938:in `block in <module:ActionView>'
/usr/local/bundle/gems/activesupport-4.2.1/lib/active_support/lazy_load_hooks.rb:38:in `instance_eval'
/usr/local/bundle/gems/activesupport-4.2.1/lib/active_support/lazy_load_hooks.rb:38:in `execute_hook'
/usr/local/bundle/gems/activesupport-4.2.1/lib/active_support/lazy_load_hooks.rb:45:in `block in run_load_hooks'
/usr/local/bundle/gems/activesupport-4.2.1/lib/active_support/lazy_load_hooks.rb:44:in `each'
/usr/local/bundle/gems/activesupport-4.2.1/lib/active_support/lazy_load_hooks.rb:44:in `run_load_hooks'
/usr/local/bundle/gems/actionview-4.2.1/lib/action_view/base.rb:203:in `<class:Base>'
/usr/local/bundle/gems/actionview-4.2.1/lib/action_view/base.rb:131:in `<module:ActionView>'
/usr/local/bundle/gems/actionview-4.2.1/lib/action_view/base.rb:10:in `<top (required)>'
/usr/local/bundle/gems/simple_form-3.1.0/lib/simple_form/action_view_extensions/form_helper.rb:69:in `<top (required)>'
/usr/local/bundle/gems/simple_form-3.1.0/lib/simple_form.rb:2:in `require'
/usr/local/bundle/gems/simple_form-3.1.0/lib/simple_form.rb:2:in `<top (required)>'
```

And this error seems to be caused by the `quality_extensions` gem which hasn't been updated to handle recent changes in Module by Rails.  See: http://stackoverflow.com/questions/25191953/aws-s3-gem-unless-defined-instance-writer-true/25878748#25878748

**Comment by jash** [comment]
This was running Rails 4.2.1.

**Comment by TylerRick** [comment]
Thanks for the bug report! This should be fixed in `thin_out_backups` 0.0.6.
